wiki/artifact-created-event-aggregation.md

```diff
@@ -1,5 +1,6 @@
-# ArtifactCreatedEvent Aggregation
+# ArtifactCreatedEvent aggregation
```

Comment:
Mattias commented earlier that titles should be capitilized, does the same apply here?